src/tests/test_generate_token.cpp

```diff
@@ -1,4 +1,4 @@
-#define BOOST_TEST_MODULE rsa
```

Comment:
Hey can you pack these changes in a separate PR?

Reply:
I'll move it to a separate PR.